### PR TITLE
Development classes

### PR DESCRIPTION
**Pull request by rocker-bell**
Awesome books code refactored using Javascript classes and methods.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello Team,

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._
- [x] Use the exact eslintrc.json config rules and also include the styleintrc.json config also.
- [x] Rename variable names and function names to reflect camelCase naming convention.
- [x] Avoid disabling the entire eslint ruleset, kindly fix the issue in the code instead.

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .eslintrc.json [hunk 1]
```diff
@@ -9,17 +9,18 @@
...
+  "extends": ["airbnb-base/legacy"],
   "rules": {
     "no-shadow": "off",
     "no-param-reassign": "off",
     "eol-last": "off",
-    "import/extensions": [ 1, {
-      "js": "always", "json": "always"
-    }]
+    
+    "max-classes-per-file": "off"
```
Comment (mikenath223) [inline .eslintrc.json]: - [ ] Kindly remove this rule and update extends to `extends: ["airbnb-base"]`. Ensure you follow the exact linting guide stated in this [link](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md). Also, add the stylelintrc.json file as stated in the link.

File: script.js [hunk 2]
```diff
@@ -1,70 +1,96 @@
...
-    document.getElementById('list_container').appendChild(x[0]).appendChild(x[1]);
-    document.getElementById('list_container').appendChild(x[0]).appendChild(x[2]);
-    document.getElementById('list_container').appendChild(x[0]).appendChild(x[3]);
-  });
+  AddBook(Book) {
+    this.BookListCollection.push(Book);
+    const TitleP = document.createElement('p');
+    const AuthorP = document.createElement('p');
+    const RButton = document.createElement('button');
+    RButton.addEventListener('click', () => { this.RemoveBook(Book.id); });
```
Comment (mikenath223) [inline script.js]: - [x] Kindly follow the camelCase naming convention and rename function and variable names to reflect this. So AddBook should be `addBook` and `this.BookListCollection` should be `this.bookListCollection`.

File: script.js [hunk 3]
```diff
@@ -1,70 +1,96 @@
...
+  RemoveBook(BookId) {
+    const BookContainer = document.getElementById(BookId);
+    BookContainer.parentNode.removeChild(BookContainer);
+    const BooksNew = [];
+    /* eslint-disable */
+    for (const i in this.BookListCollection) {
+      if (this.BookListCollection[i].id !== BookId) {
+        BooksNew.push(this.BookListCollection[i]);
+      }
+      /* eslint-enable */
```
Comment (mikenath223) [inline script.js]: - [ ] Avoid disabling the entire eslint ruleset. Please fix the issue on the code instead as that improves the code quality.

**Review by abruzy — CHANGES_REQUESTED** [review]
```markdown
Hi @rocker-bell and @mikenath223 👋🏽, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_1. Kindly fix the live demo link on your readme file._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by mimipeshy — APPROVED** [review]
## Hi Team @rocker-bell and @Abdona 👋🏾 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**